tests/quick_tests/run.cmake

```diff
@@ -61,4 +60,9 @@ recent version or use a different MPI library like MPICH.\n"
 
   ENDFOREACH()
 
+  # The messages above will not be printed if we use MESSAGE(SEND_ERROR ...)
```

Comment:
perhaps to be very clear, write `if we use MESSAGE(SEND_ERROR...) at the top` or alike so that it's clear that this applies to the first output as you actually do use `MESSAGE(SEND_ERROR)` below.